templates/site/js/usage.js

```diff
@@ -0,0 +1,39 @@
+var studentButton = document.getElementById('studentMenuButton');
+var researcherButton = document.getElementById('researcherMenuButton');
+var hobbyistButton = document.getElementById('hobbyistMenuButton');
+
+var studentUser = document.getElementById('studentUser');
+var researcherUser = document.getElementById('researcherUser');
+var hobbyistUser = document.getElementById('hobbyistUser');
+
+var displayArea = document.getElementById('getStartedDisplayArea')
+
+studentButton.style.color = "#304FFE";
+displayArea.innerHTML = studentUser.innerHTML;
+
+studentButton.onclick = function () {
+    studentButton.style.color = "#304FFE";
```

Comment:
use classes, don't assign style directly

Reply:
Changed to assign classes.